runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -588,10 +590,32 @@ private void highlightObjectsForLocation(final WorldPoint location, final int...
 		final Scene scene = client.getScene();
 		final Tile[][][] tiles = scene.getTiles();
 		final Tile tile = tiles[client.getPlane()][localLocation.getSceneX()][localLocation.getSceneY()];
+		objectsToMark.clear();
 
-		objectsToMark = Arrays.stream(tile.getGameObjects())
-			.filter(object -> object != null && ArrayUtils.contains(objectIds, object.getId()))
-			.toArray(GameObject[]::new);
+		for (GameObject object : tile.getGameObjects())
+		{
+			if (object == null)
+			{
+				continue;
+			}
+
+			for (int id : objectIds)
```

Comment:
Why not just `getImpostor()` and then check the id of that?

Reply:
Right, I guess I can do that

Reply:
Oh, that is because ObjectComposition does not have ID